[Misskey 2023.12.0-beta.1] pnpm clean-all後、Misskeyが起動しない

### 💡 Summary

`pnpm clean-all` `pnpm install` `pnpm build` `pnpm migrate`を行ったあと、`pnpm dev`でMisskeyが起動しません。

Viteによって表示されるエラーは下記のとおりです。

```
Error:   Failed to scan for dependencies from entries:
  /workspace/packages/frontend/src/_boot_.ts

  ✘ [ERROR] No loader is configured for ".json5" files: src/themes/_dark.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/_light.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/d-astro.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/d-botanical.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/d-cherry.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/d-dark.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/d-future.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/d-green-lime.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/d-green-orange.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/d-ice.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/d-persimmon.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/d-u0.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/l-apricot.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/l-botanical.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/l-cherry.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/l-coffee.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/l-light.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/l-rainy.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/l-sushi.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/l-u0.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


✘ [ERROR] No loader is configured for ".json5" files: src/themes/l-vivid.json5

    src/scripts/theme.ts:48:22:
      48 │ ...map(name => import(`../themes/${name}.json5`).then(({ default: ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~


    at failureErrorWithLog (/workspace/node_modules/.pnpm/esbuild@0.19.8/node_modules/esbuild/lib/main.js:1650:15)
    at /workspace/node_modules/.pnpm/esbuild@0.19.8/node_modules/esbuild/lib/main.js:1058:25
    at runOnEndCallbacks (/workspace/node_modules/.pnpm/esbuild@0.19.8/node_modules/esbuild/lib/main.js:1485:45)
    at buildResponseToResult (/workspace/node_modules/.pnpm/esbuild@0.19.8/node_modules/esbuild/lib/main.js:1056:7)
    at /workspace/node_modules/.pnpm/esbuild@0.19.8/node_modules/esbuild/lib/main.js:1068:9
    at new Promise (<anonymous>)
    at requestCallbacks.on-end (/workspace/node_modules/.pnpm/esbuild@0.19.8/node_modules/esbuild/lib/main.js:1067:54)
    at handleRequest (/workspace/node_modules/.pnpm/esbuild@0.19.8/node_modules/esbuild/lib/main.js:731:17)
    at handleIncomingPacket (/workspace/node_modules/.pnpm/esbuild@0.19.8/node_modules/esbuild/lib/main.js:756:7)
    at Socket.readFromStdout (/workspace/node_modules/.pnpm/esbuild@0.19.8/node_modules/esbuild/lib/main.js:679:7)
```

ここで、theme.tsの該当箇所
```typescript
	].map(name => import(`../themes/${name}.json5?import`).then(({ default: _default }): Theme => _default)),
```
のようにファイルの拡張子のあとに`?import`をつけると起動するようになります。一度これで起動すると、その後はこの差分を削除してもとに戻しても、次にクリーンするまでは起動するようです。

### 🥰 Expected Behavior

devcontainerでMisskeyが起動する

### 🤬 Actual Behavior

devcontainerでMisskeyが起動しない

### 📝 Steps to Reproduce

1. `pnpm clean` `pnpm install` `pnpm build` `pnpm migrate`を行う
2. `pnpm dev`でMisskeyを起動する

### 💻 Frontend Environment

```markdown
* Model and OS of the device(s): macOS Ventura 13.5.1
* Browser: Chrome 119.0
* Server URL: localhost
* Misskey: 2023.12.0-beta.1 (developブランチ)
```


### 🛰 Backend Environment (for server admin)

```markdown
* Installation Method or Hosting Service: development environment (devcontainer)
* Misskey: 2023.12.0-beta.1 (developブランチの12/1時点)
* Node: v20.10.0
* PostgreSQL:
* Redis:
* OS and Architecture: macOS Ventura 13.5.1
```

--- a/packages/frontend/src/scripts/theme.ts
+++ b/packages/frontend/src/scripts/theme.ts
@@ -44,7 +44,7 @@ export const getBuiltinThemes = () => Promise.all(
 		'd-cherry',
 		'd-ice',
 		'd-u0',
-	].map(name => import(`../themes/${name}.json5`).then(({ default: _default }): Theme => _default)),
+	].map(name => import(`${__dirname}/../themes/${name}.json5`).then(({ default: _default }): Theme => _default)),
 );
 
 export const getBuiltinThemesRef = () => {

